app/src/main/java/code/name/monkey/retromusic/fragments/player/md3/MD3PlaybackControlsFragment.kt

```diff
@@ -160,10 +160,8 @@ class MD3PlaybackControlsFragment :
     private fun updatePlayPauseDrawableState() {
         if (MusicPlayerRemote.isPlaying) {
             binding.playPauseButton.setImageResource(R.drawable.ic_pause_outline_small)
-            binding.playPauseCard.animateRadius(40F)
```

Comment:
The animation was kind of cool, except that the width of the button changed. But removing it would also be fine for me

Reply:
> The animation was kind of cool, except that the width of the button changed. But removing it would also be fine for me

Generally, adding some animations later to the play/pause button would be a good idea, but this animation in my point of view, along with the increase, doesn't fit the Material 3 style in any way, that's why I preferred to remove it.